cc/mcts_node.cc

```diff
@@ -413,7 +415,8 @@ void MctsNode::PruneChildren(Coord c) {
 
 std::array<float, kNumMoves> MctsNode::CalculateChildActionScore() const {
   float to_play = position.to_play() == Color::kBlack ? 1 : -1;
-  float U_scale = kPuct * std::sqrt(std::max<float>(1, N() - 1));
+  float uct = 2 * std::log((1.0f + N() + kUct_base) / kUct_base) + kUct_init;
```

Comment:
consider exact to a function or macro

Reply:
Done -- i presevered the N/N-1 behavior for children & renamed that part to 'U_common' instead of scale.